src/onpage.js

```diff
@@ -8,7 +8,7 @@
  */
 
 (function(root, doc, scriptStr, branchStr, createCallback, branchSdk, funcs, i, scriptTag, firstScript) {
-	if (!root[branchStr] || !root[branchStr]._q) {
+	if (!root[branchStr] || (root[branchStr] && !root[branchStr]._q)) {
```

Comment:
this will do exactly the same thing.

but i think the problem is, `root[branchStr]._q` is truthy when there is a branch sdk on the page AND it hasn't loaded the full script, only this shim.

what we really want is to see if any branch sdk is loaded at all. so maybe either remove the second part or make it `!root[branchStr] || root[branchStr][funcs[i]]` (to veriyf that what's on the page is actually the branch sdk)